client/blocks/comments/form.jsx

```diff
@@ -207,12 +165,12 @@ class PostCommentForm extends Component {
 					{ formTextarea }
 					<Button
 						className={ buttonClasses }
-						disabled={ this.state.commentText.length === 0 }
+						disabled={ this.props.commentText.length === 0 }
 						onClick={ this.handleSubmit }
 					>
 						{ this.props.error ? translate( 'Resend' ) : translate( 'Send' ) }
 					</Button>
-					{ this.renderError() }
+					{ error && <PostCommentFormError type={ errorType } /> }
```

Comment:
Found and filed a bug in production while testing this: https://github.com/Automattic/wp-calypso/pull/63132